build.gradle

```diff
@@ -196,17 +196,21 @@ task fastbuild(dependsOn: [
     ':dbfit-java:hsqldb:integrationTest'
     ])
 
-task integrationBuild(dependsOn: [
-    ':dbfit-java:db2:integrationTest',
-    ':dbfit-java:derby:integrationTest',
-    ':dbfit-java:hsqldb:integrationTest',
-    ':dbfit-java:mysql:integrationTest',
-    //':dbfit-java:netezza:integrationTest',
-    ':dbfit-java:oracle:integrationTest',
-    ':dbfit-java:postgres:integrationTest'
-    //':dbfit-java:sqlserver:integrationTest',
-    //':dbfit-java:teradata:integrationTest',
-    ])
+def integrationBuildTasks = []
+def excludedIntegrationBuildDbTypes = []
+
+if (project.hasProperty('excludeIntegrationTests')) {
+    excludedIntegrationBuildDbTypes=project.excludeIntegrationTests.split(",")
```

Comment:
Missing space around `=`.
split(",") seems to retain white space, new line, etc. I wonder if there is some built-in parsing which to reuse.